Sources/Rendering/Core/CellPicker/test/testCellPicker.js

```diff
@@ -64,9 +64,9 @@ test('Test vtkCellPicker image mapper', (t) => {
   const positions = picker.getPickedPositions();
   t.equal(positions.length, 1);
   const xyz = positions[0];
-  t.equal(xyz[0], 64.49344091067883);
-  t.equal(xyz[1], 75.65264519589407);
-  t.equal(xyz[2], 12);
+  t.equal(xyz[0], 64.33654020304944);
+  t.equal(xyz[1], 75.54505613923392);
+  t.equal(xyz[2], 12.000000000000004);
```

Comment:
While we expect the `resetCamera` along with added padding to the bounds to affect the x and y values, I think the z value should remain `12`. The difference is clearly just a numerical truncation error. Maybe rather than simply updating the value, I should use `floatCompare` with some tolerance?

Reply:
changed to vtkMath.areEquals test for a more stable comparison of floats.